src/api/server-routes.ts

```diff
@@ -588,17 +589,20 @@ export function setupRoutes(): express.Router {
   });
 
   router.get('/status', (req: express.Request, res: express.Response) => {
-    system.isSQLiteInstalled().subscribe(sqlite => {
-      docker.isDockerInstalled().subscribe(dockerInstalled => {
-        if (dockerInstalled) {
-          docker.isDockerRunning().subscribe(dockerRunning => {
-            const data = { sqlite: sqlite, docker: dockerInstalled, dockerRunning: dockerRunning };
+    system.isGitInstalled().subscribe(git => {
```

Comment:
please merge first three subscriptions and use `.toArray()` operator (I think)

Reply:
Hmm, I don't think merge is possible here since you need to know which one failed/succeded am I wrong? If you merge first three and you get an array `[true, true, false]` you don't know which one failed.

Reply:
use concat not merge. sorry

Reply:
tnx, yes concat will work.